.github/workflows/ruby.yml

```diff
@@ -19,17 +19,24 @@ jobs:
       matrix:
         ruby-version:
           - "2.7"
-          - "3.0"
+          #- "3.0"
+        rails-version:
+          - "5"
```

Comment:
this won't have any effect unless rails tests are ran separately and also their gems are installed separately as well

Reply:
Yep. That's part of the plan